src/ctree-app/ctree-app.html

```diff
@@ -212,21 +222,25 @@
       <app-drawer slot="drawer" id="navDrawer" class="nav-drawer" swipe-open>
         <app-toolbar>cTrees</app-toolbar>
         <div class="nav-content">
-          <paper-input label="Search" no-label-float>
+          <paper-input label="Search" value="{{searchText}}" no-label-float>
             <iron-icon icon="search" slot="prefix"></iron-icon>
+            <paper-icon-button slot="suffix" on-tap="clearSearch" icon="clear" alt="clear" title="clear"></paper-icon-button>
```

Comment:
This should only be visible if text has been entered, and it should disappear when the field is cleared.